src/shared/site-config/Header.js

```diff
@@ -12,6 +12,7 @@ const Header = () => {
             alt="Center for Strategic and International Studies"
             title="Center for Strategic and International Studies"
           />
+          <span className="site-header__logo-divider">|</span> Explore The Data
```

Comment:
Instead of including the vertical rule as an actual element in the HTML when it's only necessary for visual purposes, you can do one of two things:

1. You can set a border on the SVG & use padding & margin to position it
2. You can make this `<span className="site-header__tagline">Explore the Data</span>` and use a pseudo element to place the bar (a border won't work here because they're not the same height).